timesketch/frontend/src/components/Common/TsDynamicTable.vue

```diff
@@ -33,11 +33,44 @@ limitations under the License.
                 :numeric="typeof data[0][column.field] === 'number' ? true : false"
                 sortable
               >
+                <!-- column is an text -->
+                <div v-if="column.type === 'text'">
+                  <router-link
+                    v-if="column.searchable"
+                    :to="{ name: 'Explore', query: generateQuery(props.row[column.field], column) }"
+                  >
+                    {{ props.row[column.field] }}
+                    <i class="fas fa-search" aria-hidden="true"></i>
+                  </router-link>
+                  <span v-else>{{ props.row[column.field] }}</span>
+                </div>
+
+                <!-- column is a timestamp -->
+                <div v-else-if="column.type === 'timestamp'">
```

Comment:
is this a timestamp (seconds from epoc?) or is this a datatime?
If it is a datetime, do we want to format it somehow or just show as is?

Reply:
It depends what the provider sends, so it's hard to know in advance. We can do some more checks on the data format and attempt to translate it here, but I would leave as-is for now.

Reply:
Ack, SG.